data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -0,0 +1,44 @@
+---
+datasets:
+  - long_name: "FEMA National Risk Index"
+    short_name: "nri"
+    module_name: national_risk_index
+    last_updated_year: 2020
+    description: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
+    input_geoid_tract_field_name: "TRACTFIPS"
+    load_fields:
+      - short_name: "ex_loss"
+        df_field_name: "RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME"
+        long_name: "FEMA Risk Index Expected Annual Loss Score"
+        field_type: float
+        tile_include: true
+        csv_download: true
+        excel_download: true
+      - short_name: "ex_pop_loss"
+        df_field_name: "EXPECTED_POPULATION_LOSS_RATE_FIELD_NAME"
+        long_name: "Expected population loss rate (Natural Hazards Risk Index)"
+        field_type: float
+        tile_include: true
+        csv_download: true
+        excel_download: true
+      - short_name: "ex_ag_loss"
+        df_field_name: "EXPECTED_AGRICULTURE_LOSS_RATE_FIELD_NAME"
```

Comment:
See comment in design doc but: 

> You could just force the transform step of the ETL class to convert the field to long_name, and then run a check as part of ETL.validate to make sure all those fields are present?

Reply:
I don't understand what we're gaining here, maybe you can explain in our next pair?

Reply:
Spoke to @lucasmbrown-usds about this. 

We want to try to dynamically set (using a Python internal setter) the variables that are related to the score output and their long strings to be loaded from the YAML. @esfoobar-usds will investigate how to do this. The variables that are just internal to the dataset and are used for internal calculations should *not* be set here.

One downside could be the auto-complete on editors will not work, but @lucasmbrown-usds will comment here on how that could be solved.

Reply:
it would be something like: 

instead of: 
```
self.RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME = (
            "FEMA Risk Index Expected Annual Loss Score"
```

do something like:  
```
        self.RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME: str
        [set all these]

        for field in fields_from_config:
            setattr(self, field.df_field_name, field.long_name)

```

This way, you'd still have IDE-autocompletes on fields like self.RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME, but at least you wouldn't have to redundantly specific the long_name.

Reply:
Looks like you have to define it statically to get dynamic completion in IDEs like Pycharm: https://stackoverflow.com/a/63594033/1569221

Reply:
But maybe we don't care that much about IDE autocompletes so we don't worry about it.

Reply:
@lucasmbrown-usds and I agree that the auto-completion is not as important, but definitely would be a nice feature. We'll move forward without auto-completion for this PR.

Reply:
Done!